### DISCUSSION TITLE
How to prepare  sys_configs?

### DISCUSSION THREAD

**Discussion by McAlexDudu** [deepmodeling/dpgen#664] [Q&A] [answered]
Hi~ as I saw some examples of param.json,  sys_configs contain many POSCAR. What's the difference between them and how can I prepare them?

**Comment by AnguseZhang** [comment, marked as answer]
They serve as initial structures for MD simulations in `model_devi`.  One way is to generate via `dpgen init_bulk`. As for how it works, I would suggest refer to Figure 1 in "PHYSICAL REVIEW MATERIALS 3, 023804 (2019)" .

**Reply by McAlexDudu** [reply]
Thanks!
